hack/kiali/func-sm.sh

```
@@ -0,0 +1,369 @@
+    $(${OC} get secrets -n ${OLM_OPERATORS_NAMESPACE} cacerts --no-headers -o custom-columns=K:kind,NS:.metadata.namespace,N:.metadata.name | sed 's/  */:/g') \
+    $(${OC} get configmaps --all-namespaces --no-headers -o custom-columns=K:kind,NS:.metadata.namespace,N:.metadata.name | sed 's/  */:/g' | grep -E 'sail|servicemesh|istio')
+  do
+    local res_kind=$(echo ${r} | cut -d: -f1)
+    local res_namespace=$(echo ${r} | cut -d: -f2)
+    local res_name=$(echo ${r} | cut -d: -f3)
+    infomsg "Deleting resource [${res_name}] of kind [${res_kind}] in namespace [${res_namespace}]"
+    ${OC} delete ${res_kind} -n ${res_namespace} ${res_name}
+  done
+
+  infomsg "Delete the CRDs"
+  ${OC} get crds -o name | grep -E 'sail|servicemesh|istio' | xargs -r -n 1 ${OC} delete
```

Shellcheck (0.11.0)

[info] Double quote to prevent globbing and word splitting.

(SC2086)